src/preamble.js

```diff
@@ -1918,7 +1918,7 @@ addOnPreRun(function() {
       Runtime.loadDynamicLibrary(lib);
     });
   }
-  asm['runPostSets']();
+  asm['runPostSets'] && asm['runPostSets']();
```

Comment:
It's a hook to do relocation fixups, yeah. In a relocatable module, if memory is going to be at offset X, then you might need to write X + 12 (for a pointer into memory, and so depends on X) into X + 8 (which more obviously depends on X). So it has to be done by code.